test/compress/issue-208.js

```diff
@@ -27,3 +27,28 @@ do_update_rhs: {
         MY_DEBUG += 0;
     }
 }
+
+mixed: {
+    options = {
+        global_defs: { DEBUG: 0 }
+    }
+    input: {
```

Comment:
So this is an interesting case to consider:
```js
global_defs: { "config.DEBUG" : 1 }

var config = { engine: "voom!" };
if (config.DEBUG) {
  log();
}
```
Should we override the value and/or raise a warning, or something else altogether?